components/Activity/Charts.tsx

```diff
@@ -0,0 +1,55 @@
+import { FC } from 'react';
+import { observer } from 'mobx-react';
+import {
+  BarSeries,
+  SVGCharts,
+  Title,
+  Tooltip,
+  XAxis,
+  YAxis,
+} from 'echarts-jsx';
+import { i18n } from '../../models/Base/Translation';
+
+type ActivityDataProps = {
+  activityData: {
+    keynoteSpeechCounts: { [key: string]: number };
+    mentorOrganizationCounts: { [key: string]: number };
+  };
+};
+
+const ActivityCharts: FC<ActivityDataProps> = observer(props => {
+  const { t } = i18n;

```

Comment:
```suggestion
const { t } = i18n;

const ActivityCharts: FC<ActivityDataProps> = observer(props => {
```
`i18n` 是个单例，`t` 是它的方法，放在组件外面解构，不用每次重绘都初始化常量。